web/src/main/bvmui/src/components/Bookie.vue

```diff
@@ -1,31 +1,31 @@
 <template>
-    <v-card class="d-inline-block mx-2 mb-10" width="400" max-width="400" :color="statuscolor" raised dark @click="$emit('click', $event)">
+    <v-card class="d-inline-block mx-2 mb-10" width="400" max-width="400" :color="statusColor" raised dark @click="$emit('click', $event)">
         <v-list-item two-line>
         <v-list-item-content>
-            <v-list-item-title class="headline">{{ data.bookieId }} {{ data.clusterName }}</v-list-item-title>
-            <v-list-item-subtitle>{{ $library.formatDate(data.lastScan) }}</v-list-item-subtitle>
+            <v-list-item-title class="headline">{{ bookie.bookieId }} {{ bookie.clusterName }}</v-list-item-title>
+            <v-list-item-subtitle>{{ $library.formatDate(bookie.lastScan) }}</v-list-item-subtitle>
         </v-list-item-content>
         </v-list-item>
         <v-card-text>
-            <p class="text-center text-uppercase display-1">{{ data.state }}</p>
+            <p class="text-center text-uppercase display-1">{{ bookie.state }}</p>
         </v-card-text>
         <v-divider />
         <v-list class="transparent">
             <v-list-item>
                 <v-list-item-title>Usage</v-list-item-title>
-                <v-list-item-subtitle class="text-right">{{ $library.formatPercent(data.totalDiskSpace - data.freeDiskSpace, data.totalDiskSpace) }}&#37;</v-list-item-subtitle>
+                <v-list-item-subtitle class="text-right">{{ $library.formatPercent(bookie.totalDiskSpace - bookie.freeDiskSpace, bookie.totalDiskSpace) }}&#37;</v-list-item-subtitle>
             </v-list-item>
             <v-list-item>
                 <v-list-item-title>Endpoints</v-list-item-title>
                 <v-list-item-subtitle class="text-right">{{ data.endpoints }}</v-list-item-subtitle>
             </v-list-item>
             <v-list-item>
                 <v-list-item-title>Free space</v-list-item-title>
-                <v-list-item-subtitle class="text-right">{{ $library.formatBytes(data.freeDiskSpace) }}</v-list-item-subtitle>
+                <v-list-item-subtitle class="text-right">{{ $library.formatBytes(bookie.freeDiskSpace) }}</v-list-item-subtitle>
```

Comment:
can we open this additional information in a popup (modal dialog) ?
now we have freeDiskSpace, totalDiskSpace, endpoints and properties
this way we have far more space for the future

cc @dcdiennea